fleet_management_api/api_impl/controllers/route.py

```
@@ -156,23 +171,31 @@ def _create_empty_route_visualization(route_id: int) -> _Response:
-        return _text_response(f"Empty route visualization (route ID={route_id}) has been created.")
+        return _text_response(
+            f"Empty route visualization (route ID={route_id}) has been created."
+        )
 
 
 def _find_nonexistent_stops(*routes: _models.Route) -> _Response:
     for route in routes:
         checked_id_set: set[int] = set(route.stop_ids)
-        existing_ids = set([stop_id.id for stop_id in _db_access.get(_db_models.StopDBModel)])
+        for id_ in checked_id_set:
+            _db_access.exists(_db_models.StopDBModel, criteria={"id": lambda x: x == id_})
```

Ruff

Function definition does not bind loop variable `id_`

(B023)